frontend/src/css/components/BlogList.scss

```diff
@@ -0,0 +1,87 @@
+@import "../settings/global";
+
+.hero-section {
+    background-image: linear-gradient(#FFFDF6, #FFEBF3);
+}
+
+
+
+
+.BlogList__header{
+    
+    &__title
+    {
+        @extend h1;
+        width: 613px;
+        height: 162px;
+    }
+    
+
+    &__subtitle{
+      //  margin-top: 160px;
+        @extend h2;
+        width: 613px;
+        height: 162px;
+        
+    }   
+    &__link{
+        @extend p;
+        color:#AB1D79;
+        font-weight: bold;
+    }
+    &__img{
+      margin-top: 10px;
+  }
+
+    
+}
+  a:hover{
+      color:black;
+      text-decoration:underline;
+    }
+
+.blog_img{
+    margin-left: 400px;
+    width: 431px;
```

Comment:
the width should be 100%. It will take the width of the parent. And the parent element will have foundation class `cell large-5`. 

remove margin-left